### ISSUE TITLE
[colm@0.14.7]make[2]: Nothing to be done for 'check-am' when make check

### ISSUE THREAD

**Issue by Tom-python0121** [adrian-thurston/colm#129]
see:
```console
[root@centos8 colm]# ls
COPYING   Makefile.am  README      colm.vim  config.guess  config.status  configure     depcomp  install-sh  ltmain.sh  missing               sedsubst  test
Makefile  Makefile.in  aclocal.m4  compile   config.log    config.sub     configure.ac  doc      libtool     m4         removed_la_files.txt  src
[root@centos8 colm]# ./configure
[root@centos8 colm]# make
[root@centos8 colm]# cd test
[root@centos8 test]# ls
Makefile  Makefile.am  Makefile.in  aapl.d  colm.d  rlhc.d  rlparse.d  runtests  subject.mk  subject.mk.in  subject.sh  subject.sh.in  trans.d
[root@centos8 test]# make check
Making check in .
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test'
make[1]: Nothing to be done for 'check-am'.
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test'
Making check in aapl.d
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/aapl.d'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/aapl.d'
Making check in colm.d
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/colm.d'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/colm.d'
Making check in rlhc.d
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlhc.d'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlhc.d'
Making check in rlparse.d
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlparse.d'
make  check-am
make[2]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlparse.d'
make[2]: Nothing to be done for 'check-am'.
make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlparse.d'
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/rlparse.d'
Making check in trans.d
make[1]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/trans.d'
make  check-am
make[2]: Entering directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/trans.d'
make[2]: Nothing to be done for 'check-am'.
make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/trans.d'
make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-colm-0.14.7-rvviqroniu7qhimzpciw44vszr2l2vgq/spack-src/test/trans.d'
```
I don't know why the **check-am** doesn't work. Can you help analyze it?

**Comment by Tom-python0121** [comment]
Can I ask if I have this problem solved? Or is there something missing that caused the problem

**Comment by viccie30** [comment]
It just wasn't implemented before, but it should be fixed by 5cf37adb380b4ddf819951913c4d3044071f7e6b.
